Dockerfile

```diff
@@ -64,18 +78,31 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 # Support Gradle
 ENV TERM dumb
 ENV JAVA_OPTS "-Xms512m -Xmx1024m"
```

Comment:
we should set the max heap space higher

rule of thumb is max(1gb, 25 % of system memory)